kuma/wiki/models.py

```diff
@@ -920,6 +923,9 @@ def save(self, *args, **kwargs):
             # If this is a translation without a topic parent, try to get one.
             self.acquire_translated_topic_parent()
 
+        # Set the shortened URL (e.g. bit.ly).
+        self.set_share_url(save=False)
```

Comment:
Yeah, that would work for me, as it's the same pattern we use already for the tidied content (see `get_tidied_content`)